src/App.tsx

```diff
@@ -1,5 +1,6 @@
 import './App.scss';
 import { Routes, Route, Navigate } from 'react-router-dom';
+import React from 'react';
```

Comment:
do we need it? We have already have it in our project, in our dependencies. It could be some conflicts bcs of that.